packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -27,81 +27,69 @@
-                <ExternalLink href="https://on.cypress.io/changelog#12-0-0">
+                For a complete list of updates please review our
+                <ExternalLink href="https://on.cypress.io/changelog?utm_source=splash-page&utm_campaign=v13#13-0-0">
                   <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
-                  changelog</ExternalLink>.
+                  changelog</ExternalLink>. To take a deep-dive into this release, read our <ExternalLink href="https://on.cypress.io/cypress-13-release?utm_source=splash-page&utm_campaign=v13">
+                  blog post
+                </ExternalLink>.
               </p>
 
               <h2 class="font-bold text-[18px] text-jade-1000">
-                Testing Multi-Origin Workflows
+                Test Replay
               </h2>
 
               <p>
-                Cypress now has full support for testing multiple origins in a single test with the <ExternalLink href="https://on.cypress.io/origin">
-                  <code>cy.origin()</code>
-                </ExternalLink> command! To take a deep-dive into how this works, read our
-
-                <ExternalLink href="https://on.cypress.io/cy-origin-journey">
-                  <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
-                  blog post</ExternalLink>.
+                Test Replay brings the debugging experience you know and love from the Cypress app directly into your recorded tests in Cypress Cloud. Previously, trying to debug failures and flake in CI was painful and time consuming with only videos & screenshots. Test Replay provides a way to inspect the DOM, network events, and console logs of your application from your tests exactly as they ran in CI.
               </p>
 
-              <h2 class="font-bold mt-[24px] mb-[16px] text-[18px] text-jade-1000">
-                Test Isolation
-              </h2>
-
               <p>
-                Cypress now ensures each test runs in a clean browser context by default. We now clear the page, <code>cookies</code>, <code>localStorage</code>, and <code>sessionStorage</code> before each test to guide developers towards writing independent tests from the start.
-              </p>
-              <p>
-                If your existing tests relied on a previous test to run successfully, you might need to make some modifications to your test suite. See the
-
-                <ExternalLink href="https://on.cypress.io/migration-guide#Migrating-to-Cypress-12-0">
-                  Migration Guide
-                </ExternalLink>
-
-                for more details on what you can expect.
+                Test Replay is available in all Cypress Cloud plans. To start using Test Replay, simply record a run to Cypress Cloud. Learn more in our <ExternalLink href="https://on.cypress.io/test-replay?utm_source=splash-page&utm_campaign=v13">
+                  documentation
+                </ExternalLink>.
```

Comment:
```suggestion
                  <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
                  documentation</ExternalLink>.
```

Or we could just disable that whole. It's great for regular html, silly for rich text.